conf/nginx.conf.server

```diff
@@ -21,8 +21,8 @@ proxy_set_header  Range $slice_range;
 map $upstream_http_content_type $content_type_expiry {
     default                     1m;
     'application/json'          1m;
-    'application/pdf'           max;
-    'image/jpeg'                max;                        # max means 10 years
+    'application/pdf'           max;                        # max means 10 years to nginx
```

Comment:
Should we also include `application/octet-stream` for spreadsheets (Tables app)?

Reply:
Done!